source/accounts/index.rst

```diff
@@ -16,7 +16,17 @@ to the user community:
 - Jet
 - Orion/Hercules (MSU - external)
 
+<<<<<<< Updated upstream
```

Comment:
Looks like a bad merge happened here.  Let me know if you would like me to address this.  We can work together to help you understand how to avoid this, but I'm happy to resolve this now if that is quicker.